Src/BootCamp.Chapter/Schools/University.cs

```diff
@@ -0,0 +1,9 @@
+﻿using BootCamp.Chapter.Students;
+
+namespace BootCamp.Chapter.Schools
+{
+    public class University<TStudent> : School<TStudent> where TStudent : IStudent
```

Comment:
Can specify that this accepts only UniversityStudents.

Reply:
Fixed!